homeassistant/components/group/sensor.py

```diff
@@ -361,7 +362,7 @@ def extra_state_attributes(self) -> dict[str, Any]:
         """Return the state attributes of the sensor."""
         return {ATTR_ENTITY_ID: self._entity_ids, **self._extra_state_attribute}
 
-    @property
+    @cached_property
```

Comment:
This is an interesting one.

As it inherits the device class (intentionally) from its members. However, nothing dictates whether the members are available / loaded provided their info yet?

I do think, with the entity registry and the restore we do on start up, this is less of an issue; but it might cause issues in case the sensors used aren't available yet for older integrations.

Reply:
99893f51dd8e6c8e8e3619e77dd84e35345a6232